### PR TITLE
fix(carDealership): use the correct slug for the car buyer confirmation page

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Update confirmation pages slug

## Justify why they are needed

Confirmation pages were moved in Storyblok so we need to update accordantly.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3566** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3566?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3566?utm_source=stack-comment).

File: apps/store/src/pages/car-buyer/confirmation-with-extension/[contractId].tsx [hunk 1]
```diff
@@ -27,11 +27,12 @@ export const getServerSideProps: GetServerSideProps<Props, Params> = async (cont
   const apolloClient = await initializeApolloServerSide({ req, res, locale })
   // with-extension | without-extension
   const confirmationStorySlug = resolvedUrl.split('/')[2]
+  const slug = `car-buyer/${confirmationStorySlug}`
 
   const [trialExtension, layoutWithMenuProps, story] = await Promise.all([
     fetchTrialExtensionData(apolloClient, contractId),
     getLayoutWithMenuProps(context, apolloClient),
-    getStoryBySlug<ConfirmationStory>(confirmationStorySlug, { locale }),
+    getStoryBySlug<ConfirmationStory>(slug, { locale, version: 'draft' }),
```
Comment (robinandeer) [inline apps/store/src/pages/car-buyer/confirmation-with-extension/[contractId].tsx:35]: this seems like a bug -- we should not always get draft version, right?

**Review by robinandeer — COMMENTED** [review]
t

**Review by gustaveen — APPROVED** [review]
Perfect! 🙌

File: apps/store/src/pages/car-buyer/confirmation-with-extension/[contractId].tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/pages/car-buyer/confirmation-with-extension/[contractId].tsx:35]: God damn it. Yes, that's a mistake. Thanks

**Pull request merged**